src/main/java/com/mateacademy/sql/ProjectTable.java

```diff
@@ -0,0 +1,48 @@
+package com.mateacademy.sql;
+
+
+import lombok.Getter;
+
+@Getter
+public class ProjectTable {
+    private String name;
+    private Integer developersNumber;
+    private String customer;
+    private Integer price;
+    private Integer cost;
+    private String creationDate;
+
+    ProjectTable(String name, Integer developersNumber, String customer, Integer price, Integer cost, String creationDate) {
+        this.name = name;
+        this.developersNumber = developersNumber;
+        this.customer = customer;
+        this.price = price;
+        this.cost = cost;
+        this.creationDate = creationDate;
+    }
+
+    public String getName() {
```

Comment:
Why you add getters. You already have  Lombok annotation @Getter

Reply:
@getter doesn't work, idk why

Reply:
Did you install the Lombok plugin?

Reply:
I added lombok dependency to pom.xml

Reply:
To process Lombok annotation you need Lombok plugin.